python/lsst/obs/pfs/constructFiberFlatTask.py

```diff
@@ -188,9 +188,14 @@ def combine(self, cache, struct):
         sumVariances = drpStella.where(sumVariances, '<=', 0., 0.1, sumVariances)
         snrArr = sumFlats / np.sqrt(sumVariances)
 
-        sumFlats = drpStella.where(sumFlats, '<=', 0., 0.1, sumFlats)
-        normalizedFlat = sumRecIm.getArray() / sumFlats
-        normalizedFlat = drpStella.where(sumRecIm.getArray(), '<=', 0., 1., normalizedFlat)
+        sumRecImArr = sumRecIm.getArray()
+        sumRecImArr = np.where(sumRecImArr <= 0.,
+                               0.1,
```

Comment:
Changed to 0.01 and added comment:
```
to avoid division by zero and remove non-physical negative values,
we set the reconstructed values <= 0.0 to 0.01. We later set the normalized
Flat to unity where sumRecImArr <= 0.01, as well as all other pixels
with a SNR < self.config.minSNR
```